R/scale-gradient.r

```diff
@@ -134,3 +134,24 @@ scale_fill_gradientn <- function(..., colours, values = NULL, space = "Lab", na.
   continuous_scale(aesthetics, "gradientn",
     gradient_n_pal(colours, values, space), na.value = na.value, guide = guide, ...)
 }
+#' @rdname scale_gradient
+#' @export
+scale_colour_steps <- function(..., low = "#132B43", high = "#56B1F7", space = "Lab",
```

Comment:
Why isn't this scale colour binned?

Reply:
It follows the style of `scale_colour_gradient` that gets defined and then is assigned to `scale_colour_continuous`